scapy/dadict.py

```diff
@@ -25,7 +25,7 @@ class DADict_Exception(Scapy_Exception):
 class DADict:
     def __init__(self, _name="DADict", **kargs):
         self._name=_name
-        self.__dict__.update(kargs)
+        self._update(kargs)
```

Comment:
That's a fix for a bug that would be trigger by this PR:
As you can see, `__setitem__` uses the `fixname` function before setting the items. But for a reason, using `self.__dict__.update` do not use `__setitem__`, so what was provided in `__init__` was not fixed.